CAP2PROP.md

```diff
@@ -1 +1,41 @@
 # Capstone Project #2 Proposal
+
+**Project Overview**: Proposal is to build a predictive model for a "hypothetical" movie that can be user defined across a defined set of critical features such as leading actors/actresses, genre, and plot descriptions. This is an interesting problem for a number of reasons:
+
+- There are a large number of potential variables that can explain the success (or lack thereof) of a movie, providing the model building a significant challenge in identifying and staging the most relevant variables
+- The problem is highly relatable to a large number of people and can be turned into an engaging application as long as the features that can be used to define the "hypothetical" movie are simple in nature but comprehensive in coverage. In addition, the application can be used in a "live" environment (e.g. to predict the box office revenues of yet to be released movies, and then tracking actual vs. predicted revenues in real time)
```

Comment:
Are you envisioning a UI where users can change movie attributes and get "predicted" box office performance?